options.go

```diff
@@ -176,6 +177,15 @@ func WithHasher(hasher func() hash.Hash32) ProcessorOption {
 	}
 }
 
+// WithHotStandby configures the processor to keep up partitions that it is not assigned when using
+// multiple processors in a group. This will use more resources, but allows to fail over processors in case of crashes
```

Comment:
This sounds like failover is impossible without hotstandby. I think what you want to say here is something like "... in a group. WithHotStandby allows fast processor failover since all partitions are hot in other processor instances, but it requires more resources (eg, network and disk)."

Reply:
thanks diogo, updated the comment

Reply:
fixed